application/common/models/Password.php

```diff
@@ -19,7 +19,11 @@
  */
 class Password extends PasswordBase
 {
-    const SCENARIO_UPDATE_METADATA = 'update_metadata';
+    public const SCENARIO_UPDATE_METADATA = 'update_metadata';
+
+    public const HASH_ALGORITHM = PASSWORD_BCRYPT;
```

Comment:
It may be worth adding a comment here that, if we ever change this to something other than `PASSWORD_BCRYPT`, we will need to change (potentially) all of the places in this file where we use PHP's built-in `password_*()` functions, because the options are different for different hashing algorithms (as you probably figured out before I did).

Reply:
Ick. I'll make another pass at this.